<<<<<<< HEAD
# Service to Set the max Speed of the robot.

float64 tcp  # mm/s
float64 ori  # deg/s
---
int64 ret
string msg
=======
# Service to Set the max Speed of the robot.

float64 tcp  # mm/s
float64 ori  # deg/s
---
int64 ret
string msg
>>>>>>> 1b1f87b6247333221ca1660d810eda01b82fd03f
